<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="lenovo-t450" ignored="0" name="Assertion Test Case" tests="3" failures="2" timestamp="2021-07-09T15:11:21 IST" time="0.066" errors="0">
  <testcase name="LoginRediffmail" time="0.014" classname="annotations.AssertionsTestAnnotation">
    <failure type="java.lang.AssertionError" message="Actual is not equal to expected expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Actual is not equal to expected expected [true] but found [false]
at annotations.AssertionsTestAnnotation.LoginRediffmail(AssertionsTestAnnotation.java:62)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- LoginRediffmail -->
  <testcase name="SendMail" time="0.001" classname="annotations.AssertionsTestAnnotation"/>
  <testcase name="Logout" time="0.003" classname="annotations.AssertionsTestAnnotation">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Actual is equal to expected - Logout Test annotation method did not expect [true] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Actual is equal to expected - Logout Test annotation method did not expect [true] but found [true]
at annotations.AssertionsTestAnnotation.Logout(AssertionsTestAnnotation.java:98)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Logout -->
</testsuite> <!-- Assertion Test Case -->
